GG/GG/GUI.h

```diff
@@ -6,12 +6,12 @@
    modify it under the terms of the GNU Lesser General Public License
    as published by the Free Software Foundation; either version 2.1
    of the License, or (at your option) any later version.
-   
```

Comment:
In the midst of a bunch of unrelated changes, it's distracting and arbitrary seeming.